pkg/util/mount/mount_linux.go

```diff
@@ -921,6 +921,14 @@ func (mounter *Mounter) SafeMakeDir(pathname string, base string, perm os.FileMo
 	return doSafeMakeDir(pathname, base, perm)
 }
 
+func (mounter *Mounter) KubeletPath(pathname string) (string, error) {
+	realpath, err := filepath.EvalSymlinks(pathname)
```

Comment:
Those codes are exactly the same on Linux and Windows. It is better to move to a common function and call it here.

Reply:
This method will be [removed finally](https://github.com/kubernetes/kubernetes/pull/62903#discussion_r183209701).